capture-sdk/sdk/src/main/java/net/gini/android/capture/noresults/NoResultsActivity.java

```diff
@@ -43,96 +43,40 @@
  *     <ul>
  *         <li>
  *             <b>Header icon:</b> via images for mdpi, hdpi, xhdpi, xxhdpi, xxxhdpi named
- *             {@code gc_alert_icon.png}
```

Comment:
[Request] Can you please add `Internal use only.` at the top of the javadoc for `NoResultsActivity`? In the new version we don't want clients to use the activity directly anymore.